lib/Controller/DisplayProfile.php

```diff
@@ -305,10 +310,18 @@ public function editForm($displayProfileId)
     {
         // Create a form out of the config object.
         $displayProfile = $this->displayProfileFactory->getById($displayProfileId);
+        // Get the Player Version for this display profile type
+        $playerVersions  = $this->playerVersionFactory->query(null, ['playerType' => $displayProfile->type]);
```

Comment:
I thought our edit form used a select2 to look up this data (calling GET /playersoftware) ?

Reply:
We do indeed, but I think I need to query it here (and similar in Display controller) and pass versions to the twig as a "field.options" and  with the field.value in select 2 dropdown, to have correct default / previously selected version correctly displayed there?

Reply:
You only need to add 1 item to it, which is the currently selected one (if there is one). So if you have a getByMediaId in your playerVersionFactory you could use that, along with the `$display->getSetting('versionMediaId', <only get the overrides>);`

Imagine that there are 100 versions - the above would load all of those into a list, and never show them (because as soon as you open the list, the query fires to get the paged version).